src/components/shared/LeftSideMenu/index.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styles from './leftSideMenu.module.scss';
+
+const LeftSideMenu = () => (
+  <div className={styles.container}>
+    <div className={styles.block}>
+      <Link to="/admin/home">
+        <div className={styles.item}>
+          <img src="/images/home.png" alt="home page" />
+        </div>
+      </Link>
+    </div>
+    <div className={styles.block}>
+      <Link to="/admin/databases">
+        <div className={styles.item}>
+          <img src="/images/database.png" alt="databases page" />
+        </div>
+      </Link>
+
+    </div>
+    <div className={styles.block}>
+      <Link to="/admin/models">
+        <div className={styles.item}>
+          <img src="/images/spreadsheet.png" alt="models page" />
+        </div>
+      </Link>
+
+    </div>
+    <div className={styles.block}>
+      <Link to="/admin/users">
+        <div className={styles.item}>
+          <img src="/images/group.png" alt="users page" />
+        </div>
+      </Link>
+
+    </div>
```

Comment:
I don't really like such kinds of code repeat
Maybe it would be better to move all info to array of objects, like:
```
[
  {
    style: ...,
    link: '',
  }
]
```
And create a map loop
But as you wish